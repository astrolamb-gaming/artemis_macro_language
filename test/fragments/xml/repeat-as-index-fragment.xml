<mission_data version="1.0">
   <start>
    <repeat _length="5" _as="x" >
        <repeat _length="3" _as="y" >
            <set_variable name="m_${x}_x_${y}" value="${_.float(x)*_.float(y)}" />
        </repeat>
    </repeat>
  </start>
</mission_data>