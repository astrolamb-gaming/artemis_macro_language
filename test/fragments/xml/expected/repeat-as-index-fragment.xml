<mission_data version="1.0">
  <start>
    <set_variable name="m_0_x_0" value="0"/>
    <set_variable name="m_0_x_1" value="0"/>
    <set_variable name="m_0_x_2" value="0"/>
    <set_variable name="m_1_x_0" value="0"/>
    <set_variable name="m_1_x_1" value="1"/>
    <set_variable name="m_1_x_2" value="2"/>
    <set_variable name="m_2_x_0" value="0"/>
    <set_variable name="m_2_x_1" value="2"/>
    <set_variable name="m_2_x_2" value="4"/>
    <set_variable name="m_3_x_0" value="0"/>
    <set_variable name="m_3_x_1" value="3"/>
    <set_variable name="m_3_x_2" value="6"/>
    <set_variable name="m_4_x_0" value="0"/>
    <set_variable name="m_4_x_1" value="4"/>
    <set_variable name="m_4_x_2" value="8"/>
  </start>
</mission_data>